sys/zuse.hoon

```diff
@@ -8022,7 +8022,7 @@
       %+  turn  (gulf 0 255)
       |=  gal=@
       %+  request-to-json:ethereum
-        `(scot %ud gal)
```

Comment:
Yes, and perhaps this should be in comments somewhere.  Basically, if parity can parse an id as a number, then it returns it as a JSON number.  Geth just spits it back to you how you gave it.  I've opted to force everything into strings, but I *think* we could also resolve this by making them real numbers and parsing them as such.

Reply:
Also, I'll try to be better about making commits be about one thing at a time.  As a workflow, I work on several problems at once so that I can keep working during the long build cycle, but that shouldn't hit the git history.

Reply:
I don't have any objection to stringifying these. Could you update the tests in `tests/sys/zuse/dawn.hoon`? You can run them specifically with `+test /sys/zuse/dawn`. (for some reason, the CI runner has stopped detecting test failures, which is kind of counter-productive ...)

Reply:
> but I think we could also resolve this by making them real numbers and parsing them as such.

As far as the RPC stdlib stuff goes, we probably don't want to do that, since some apps may want to use this with text IDs still. (otoh, a `@t` is a perfectly valid number. (^: )  
As-is, this is a fair workaround for an awful problem to have.

> Also, I'll try to be better about making commits be about one thing at a time.

Thank you! That's my workflow too, so I know separating changes properly can be a pain sometimes. The effort is much appreciated though! (: